### ISSUE TITLE
Recover from network connectivity or server issues automatically

### ISSUE THREAD

**Issue by leosh64** [jan-molak/jenkins-build-monitor-plugin#460]
If the build monitor plugin page encounters an error with the connection to the Jenkins host (e.g. network issue such as host temporarily unreachable, or Jenkins server error 403), the entire screen is greyed out and a warning popup is shown with the error message.

While this is generally a nice functionality, it would be great if the build monitor could try to automatically recover from this state, for example:

-  try to query the server again after a certain time, and 
-  if successful, hide the warning popup and resume normal operation

**Comment by CendioOssman** [comment]
This is extra critical for a function like this since it is generally used on a screen with no keyboard or mouse available. So there is no way to manually refresh the page to recover.

**Comment by leosh64** [comment]
> This is extra critical for a function like this since it is generally used on a screen with no keyboard or mouse available. So there is no way to manually refresh the page to recover.

Exactly. Currently, we always have to restart the edge client displaying the build monitor page.

**Comment by LeoSko** [comment]
So I've got some stupid solution to this, but that works. Via Tampermonkey extension just integrate this pastebin: https://pastebin.com/JyibgWgD

It basically checks for 2 things every 5 seconds:
* if there are any buttons `Refresh`, it clicks them - in case mothership dies
* if there are any `h1`s with `Sorry to bother you,` text, it reloads the page - in case e.g. 503 is returned
